javascript/javascript1/week2/CandyHelperOptional.js

```diff
@@ -0,0 +1,50 @@
+const boughtCandyPrices = [];
+let sweet, 
```

Comment:
I like the approach but not fully sure that it works correctly. 
all variables  like sweet, chocolate, toffee are not initialised. It means that all of them have the same value === undefined. 
This means that when candy is added
e.g. 
addCandy(toffee, 10) we start checking if/else-ifs conditions. 
Since toffee === undefined and sweet === undefined
the first condition in `if` works and we are counting price as for `sweet` instead of `toffee`